themes/sbvfrd/ts/app.ts

```diff
@@ -15,5 +22,14 @@ ${p.firstName}&amp;type=Author" title=" ${p.lastName}, ${p.firstName}">${p.lastN
 authorid="${p["@id"]}"></span></li>`;
     };
 
-    client.renderContributors(recordIdEl.value, list, template);
+    if (recordIdEl) {
```

Comment:
Condition might not be sufficient. We are going to refactor this later.

Reply:
Might be a good idea, as I saw after reading your comment that on landing on a single entry page where the recordIdEl is available some error occurs due to server response 500.